packages/lodestar/src/constants/network.ts

```diff
@@ -37,34 +39,56 @@ export enum MethodResponseType {
 export const Methods = {
   [Method.Status]: {
     requestSSZType: (config: IBeaconConfig): typeof config.types.Status => config.types.Status,
-    responseSSZType: (config: IBeaconConfig): typeof config.types.Status => config.types.Status,
+    responseSSZType: (config: IBeaconConfig): {type: ContainerType<Status>; isTreeBacked: boolean} => ({
+      type: config.types.Status,
+      isTreeBacked: false,
+    }),
     responseType: MethodResponseType.SingleResponse,
   },
   [Method.Goodbye]: {
     requestSSZType: (config: IBeaconConfig): typeof config.types.Goodbye => config.types.Goodbye,
-    responseSSZType: (config: IBeaconConfig): typeof config.types.Goodbye => config.types.Goodbye,
+    responseSSZType: (config: IBeaconConfig): {type: BigIntUintType; isTreeBacked: boolean} => ({
+      type: config.types.Goodbye,
+      isTreeBacked: false,
+    }),
     responseType: MethodResponseType.SingleResponse,
   },
   [Method.Ping]: {
     requestSSZType: (config: IBeaconConfig): typeof config.types.Ping => config.types.Ping,
-    responseSSZType: (config: IBeaconConfig): typeof config.types.Ping => config.types.Ping,
+    responseSSZType: (config: IBeaconConfig): {type: BigIntUintType; isTreeBacked: boolean} => ({
+      type: config.types.Ping,
+      isTreeBacked: false,
```

Comment:
Can't we determine backing from value directly instead of passing separate boolean flags?

Reply:
@wemeetagain could you help with this :)